src/jdk.internal.vm.ci/share/classes/jdk.vm.ci.hotspot/src/jdk/vm/ci/hotspot/HotSpotVMConfig.java

```diff
@@ -152,7 +151,7 @@ String getHostArchitectureName() {
     /**
      * See {@code markWord::prototype()}.
      */
-    long arrayPrototypeMarkWord() {
+    long prototypeMarkWord() {
         return markWordNoHashInPlace | markWordNoLockInPlace;
     }
```

Comment:
It is not immediately obvious that this is correct for all object types. Does this match what the initial_mark() is now?

Reply:
Yes, with biased locking gone there is now only one prototype for the markword. This should match markWord::prototype().